src/main/java/frc/robot/commands/GoHomeSequence.java

```diff
@@ -0,0 +1,37 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.ConditionalCommand;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
+import frc.robot.subsystems.elevator.Elevator;
+import frc.robot.subsystems.elevator.Elevator.ElevatorPosition;
+import frc.robot.subsystems.intake.Intake;
+import frc.robot.subsystems.intake.Intake.IntakePosition;
+
+public class GoHomeSequence extends SequentialCommandGroup {
+  public GoHomeSequence(Intake intake, Elevator elevator) {
+    addRequirements(intake, elevator);
+    addCommands(
+        new ConditionalCommand(
```

Comment:
Separate out the `SequentialCommandGroup`s into their own named commands, for readability.